ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +0,0 @@
-m_EditorVersion: 2018.2.11f1
```

Comment:
Any reason why do we need this change?

Reply:
This is generated by build script, we don't need to keep this one. This is changed based on Unity IDE used on a machine so it can be changed at anytime. Considering this as user data.